### PR TITLE
feat: improve virtual text highlight when 'cursorline' set

### PR DESCRIPTION
**Pull request by luozhiya**
Hi,

This PR set virtual text highlight mode to `combine`.

Before
image

After
image

### CODE REVIEW

**Review by andersevenrud — APPROVED** [review]
Nice!

File: lua/nvim_context_vt/utils.lua [hunk 1]
```diff
@@ -78,7 +78,7 @@ M.create_virtual_text_factory = function(parser, ft, opts)
     local function text_from_node(node)
         local vt = parser(node, ft, opts)
         if vt then
-            return { virt_text = { { vt, opts.highlight } } }
+            return { virt_text = { { vt, opts.highlight } }, hl_mode = 'combine', }
```
Comment (andersevenrud) [inline lua/nvim_context_vt/utils.lua]: Could you remove the trailing comma ?

File: lua/nvim_context_vt/utils.lua [hunk 2]
```diff
@@ -91,7 +91,7 @@ M.create_virtual_text_factory = function(parser, ft, opts)
 
             local lines = lines_from_nodes(nodes)
             if #lines > 0 then
-                return { virt_lines = lines }
+                return { virt_lines = lines, hl_mode = 'combine', }
```
Comment (andersevenrud) [inline lua/nvim_context_vt/utils.lua]: Could you remove the trailing comma ?

**Review by andersevenrud — CHANGES_REQUESTED** [review]
In hindsight, there's a few spots that the linter that didn't run would probably complain about.

**Comment by luozhiya** [comment]
@andersevenrud 

> remove the trailing comma

Fixed, Rebased

**Comment by andersevenrud** [comment]
Thanks!

**Pull request merged**